lib/ReactViews/HelpScreens/HelpPanel.jsx

```diff
@@ -0,0 +1,151 @@
+'use strict';
+
+import React from 'react';
+import ObserveModelMixin from '../ObserveModelMixin';
+import defined from 'terriajs-cesium/Source/Core/defined';
+import classNames from 'classnames';
+import MenuPanel from '../StandardUserInterface/customizable/MenuPanel.jsx';
+
+import Styles from './help-panel.scss';
+import DropdownStyles from '../Map/Panels/panel.scss';
+import helpIcon from '../../../wwwroot/images/icons/help.svg';
+
+const HelpPanel = React.createClass({
+    mixins: [ObserveModelMixin],
+
+    propTypes: {
+        viewState: React.PropTypes.object.isRequired,
+        helpSequences: React.PropTypes.object
+    },
+
+    componentWillMount() {
+        const that = this;
+        setInterval(function() {
```

Comment:
Ideally you would only set the interval when you have screens to show.  A fast interval like this one (10ms!) will run even when you're focused on another tab and will have implications on battery life.